grails-app/services/au/org/ala/bie/ImportService.groovy

```diff
@@ -1796,7 +1796,7 @@ class ImportService implements GrailsConfigurationAware {
                 String imageId = getImageFromParamList(preferredImagesList, doc.guid)
                 log.info ("Updating: guid " + doc.guid + " with imageId " + imageId)
                 if (!doc.containsKey("image") || (doc.containsKey("image") && doc.image != imageId)) {
-                    updateImage(doc, imageId, buffer, online)
```

Comment:
Did the method have a `online` argument?

Reply:
No. This is meant to just update the online index, so online should always be true. The online variable was a bug.

Reply:
no wonder I couldn't find it :)